src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -1381,7 +1374,7 @@ namespace Akka.Actor
     }
     public class RepointableActorRef : Akka.Actor.ActorRefWithCell, Akka.Actor.IActorRefScope, Akka.Actor.IRepointableRef
     {
-        public RepointableActorRef(Akka.Actor.Internal.ActorSystemImpl system, Akka.Actor.Props props, Akka.Dispatch.MessageDispatcher dispatcher, System.Func<Akka.Dispatch.Mailbox> createMailbox, Akka.Actor.IInternalActorRef supervisor, Akka.Actor.ActorPath path) { }
+        public RepointableActorRef(Akka.Actor.Internal.ActorSystemImpl system, Akka.Actor.Props props, Akka.Dispatch.MessageDispatcher dispatcher, Akka.Dispatch.MailboxType mailboxType, Akka.Actor.IInternalActorRef supervisor, Akka.Actor.ActorPath path) { }
```

Comment:
Same change: no more mailbox producer functions in any `IActorRef` constructors. This is all done via `MailboxType` now.